test.bats

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bats
```

Comment:
TIL there is a bash unit testing framework :)

Reply:
BATS ftw!